packages/eslint-plugin/tests/rules/explicit-member-accessibility.test.ts

```diff
@@ -1392,13 +1442,15 @@ class Test {
           messageId: 'missingAccessibility',
           line: 3,
           column: 3,
+          endLine: 3,
+          endColumn: 14,
           suggestions: [
             {
               messageId: 'addExplicitAccessibility',
               data: { type: 'public' },
               output: `
 class Test {
-  public constructor(public ...x: any[]) {}
+  public constructor(public x: any[]) {}
```

Comment:
changed this since it's illegal syntax for a parameter property to be a rest element. Though the rule doesn't really care.